pages/en/snapps/how-to-write-a-snapp.mdx

```diff
@@ -387,3 +387,10 @@ This is not yet production ready, but is work in progress.
 
 </Alert>
 
+## What's next? 
+
+Now that you've made it this far, you're now ready to start a project!
+
+If you run into any questions or want to discuss snapps with other developers, join the [#snapp-developers](https://discord.gg/654BcVJf) channel on Discord. 
+
+If you want to see all resources related to snapps, including community built snapps, check out the [Snapp Developer Resource Kit](https://github.com/o1-labs/snapp-resource-kit).
```

Comment:
> If you'd like to view more resources related to snapps, including community-built snapps, check out the [Snapp Developer Resource 
> Kit](https://github.com/o1-labs/snapp-resource-kit).

Minor punctuation fix + suggesting a slight rephrase to avoid implying it's the canonical source.